docs/source/start.rst

```diff
@@ -156,73 +155,14 @@ libffi. On a Debian-derivative system, this could be accomplished with
 To uninstall, remove the directory ``~/rigetti``.
 
 
-Using the SDK
-~~~~~~~~~~~~~
-
-The SDK may either be used directly from the command line, or through PyQuil. Refer to the manual pages for the QVM and
-compiler for information on how to use them directly. One can read the manual pages by open a new terminal window and
+Using the QVM and Compiler
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+The QVM and the Quil compiler may either be used through pyQuil, or directly from the command line. Refer to the manual pages for
+each of these for information on how to use them directly. After installation, one can read the manual pages by opening a new terminal window and
 typing ``man qvm`` (for the QVM) or ``man quilc`` (for the compiler). One can quit out of the manual page by typing ``q``.
 
-Connecting to the SDK
```

Comment:
This doesn't really belong here. Users running locally don't need these files, and users on the QMI have these files pre-configured. This should be ported to another page specifically dedicated to explaining these files sometime this week.